### PR TITLE
docs: add forward-auth zh document

### PR DESCRIPTION
**Pull request by biubiue**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/plugins/forward-auth.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+| upstream_headers | array[string] | 可选 |  |  | 认证通过时，设置 `authorization` 服务转发至 `upstream` 的请求头。如果不设置则不转发任何请求头。
+| client_headers | array[string] | 可选 |  |  | 认证失败时，由 `authorization` 服务向 `client` 发送的响应头。如果不设置则不转发任何响应头。 |
+| timeout | integer | 可选 | 3000ms | [1, 60000]ms | `authorization` 服务请求超时时间 |
+| keepalive | boolean | 可选 | true |  | HTTP 长连接 |
+| keepalive_timeout | integer | 可选 | 60000ms | [1000, ...]ms | 长连接空闲时间 |
+| keepalive_pool | integer | 可选 | 5 | [1, ...]ms | 连接池大小 |
+
+## 数据定义
+
+request_headers属性中转发到 `authorization` 服务中的APISIX内容清单
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:55]:
```suggestion
request_headers 属性中转发到 `authorization` 服务中的 Apache APISIX 内容清单
```
There should be spaces between Chinese and English.

**Review by tzssangglass — APPROVED** [review]

File: docs/zh/latest/plugins/forward-auth.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+| upstream_headers | array[string] | 可选 |  |  | 认证通过时，设置 `authorization` 服务转发至 `upstream` 的请求头。如果不设置则不转发任何请求头。
+| client_headers | array[string] | 可选 |  |  | 认证失败时，由 `authorization` 服务向 `client` 发送的响应头。如果不设置则不转发任何响应头。 |
+| timeout | integer | 可选 | 3000ms | [1, 60000]ms | `authorization` 服务请求超时时间 |
+| keepalive | boolean | 可选 | true |  | HTTP 长连接 |
+| keepalive_timeout | integer | 可选 | 60000ms | [1000, ...]ms | 长连接空闲时间 |
+| keepalive_pool | integer | 可选 | 5 | [1, ...]ms | 连接池大小 |
+
+## 数据定义
+
+request_headers属性中转发到 `authorization` 服务中的 Apache APISIX 内容清单
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:55]:
```suggestion
request_headers 属性中转发到 `authorization` 服务中的 Apache APISIX 内容清单
```

Comment (biubiue) [reply docs/zh/latest/plugins/forward-auth.md]: It’s ok

**Review by zaunist — APPROVED** [review]
LGTM

File: docs/zh/latest/plugins/forward-auth.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+3. **client_headers** 当授权失败时转发 `authorization` 服务响应头到 `client`
+
+```shell
+curl -i http://127.0.0.1:9080/headers
+HTTP/1.1 403 Forbidden
+Location: http://example.com/auth
+```
+
+最后,你可以通过在路由中移除的方式禁用 `forward-auth` 插件。
```
Comment (spacewander) [inline docs/zh/latest/plugins/forward-auth.md:140]:
```suggestion
最后，你可以通过在路由中移除的方式禁用 `forward-auth` 插件。
```

**Review by spacewander — APPROVED** [review]

**Pull request merged**